templates/flavors/kubeadm/default/kustomization.yaml

```diff
@@ -4,7 +4,6 @@ resources:
   - ../../../infra
   - kubeadmConfigTemplate.yaml
   - kubeadmControlPlane.yaml
-  - allow-etcd-policy.yaml
```

Comment:
I think we have some docs noting that this is being set, could you also update to reflect that this is no longer added through a CCNP in our docs?

Reply:
Updated it. We do require this for kubeadm vpcless flavors as etcd there still uses public ip. Updated docs to reflect the same.